src/Dotnet.Script/Program.cs

```diff
@@ -133,17 +133,20 @@ private static int Wain(string[] args)
                 c.Description = "Creates an executable from a script";
                 var fileNameArgument = c.Argument("filename", "The script file name");
                 var publishDirectoryOption = c.Option("-o |--output", "Directory where the published executable should be placed.  Defaults to a 'publish' folder in the current directory.", CommandOptionType.SingleValue);
+                var dllOption = c.Option("--dll", "Publish to a .dll instead of a .exe", CommandOptionType.NoValue);
+                var commandConfig = c.Option("-c | --configuration <configuration>", "Configuration to use for running the script [Release/Debug] Default is \"Debug\"", CommandOptionType.SingleValue);
                 var publishDebugMode = c.Option(DebugFlagShort + " | " + DebugFlagLong, "Enables debug output.", CommandOptionType.NoValue);
                 c.OnExecute(() =>
                 {
+                    var x = debugMode.HasValue();
```

Comment:
what's `x`? 😀

Reply:
I hate that visual studio didn't give a warning for this type of unused variable...